src/components/styles/BookList.css

```diff
@@ -0,0 +1,179 @@
+.books-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #f5f6fa;
+  gap: 2rem;
+  min-height: 100vh;
+}
+
+section {
+  width: 100%;
+  border-radius: 4px;
+  border: 1px solid #e8e8e8;
+  background-color: white;
+  display: flex;
+  flex-direction: row;
+  transition: transform 0.3s, background-color 0.3s;
+  margin-top: 20px;
+}
+
+section:hover {
+  transform: scale(1.1);
+  background-color: #f5f6fa;
+}
+
+ul {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+}
+
+li {
+  width: 100%;
+  margin-bottom: 10px;
+}
+
+li:nth-child(1) {
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+}
+
+li:nth-child(2) {
+  height: 1.813rem;
+  margin: 0.188rem -10rem 50px 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
+}
+
+li:nth-child(3) {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
```

Comment:
- [ ] Well done adding styling to your project 😊Although I noticed so many repeated rules in your CSS. 

- [ ] In our stylesheets, let's ensure that basic CSS bad practices are avoided ([article for support](https://speckyboy.com/good-bad-css-practices))

- [ ] To follow our list of the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

### No repeating of the same rule

- [ ] If we want to create a CSS rule for multiple elements, we use a class attribute instead. [example](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Please correct all occurrences 🛠